### DISCUSSION TITLE
Use Tapping Keys To Confirm Pause Selection

### DISCUSSION THREAD

**Discussion by Fr3akShvt** [ppy/osu#29960] [Ideas] [answered]
I've done this for years in standard and one of the main issues I have with laser is the inability to confirm a selection with my tapping keys. For example, if I paused the game and wanted to go back to song select I wouldn't be able to hover over the option and press x for example to confirm it. I know it's something simple but would be a huge QOL improvement for me and a lot of people I know who do the same thing.

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue: https://github.com/ppy/osu/issues/3551
